rapidtide/workflows/rapidtide_parser.py

```diff
@@ -46,7 +46,6 @@ def _get_parser():
     parser = argparse.ArgumentParser(
         prog="rapidtide",
         description=("Perform a RIPTiDe time delay analysis on a dataset."),
-        usage="%(prog)s in_file outputname [options]",
```

Comment:
Is it cool to remove this? I don't know if you want the arguments to show up in the help string, but the patterns in that section are useful for indicating mutually exclusive arguments.

Reply:
Yes, that's fine.  I think that's better (didn't know it would do that!)

Reply:
Awesome! I think this PR is ready for review then. That was the only change that I wasn't sure you'd want, but please let me know if there's anything in the rest of the PR you want me to change.

Reply:
So I generated a new theoryofoperation.rst after we deleted the one that was there.  This PR won't overwrite that, correct?

Reply:
I didn't delete `theoryofoperation.rst` in this PR, so there won't be a conflict.

Reply:
👍